core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -418,6 +422,10 @@ public static String textDistributedPlan(
                 .collect(toImmutableMap(DynamicFilterDomainStats::getDynamicFilterId, identity()));
         TypeProvider typeProvider = getTypeProvider(allFragments);
 
+        if (verbose) {
```

Comment:
Why not print this always ?

Reply:
I would say `EXPLAIN ANALYZE` is more for end-users while `EXPLAIN ANALYZE VERBOSE` it's kind of diagnostic output for devs and power users. Therefore, I thought it's not that useful to keep it in plain `EXPLAIN ANALYZE`

Reply:
Why not always?

every EXPLAIN is quite verbose, and version string is quite short, so having it in every EXPLAIN & EXPLAIN ANALYZE wouldn't hurt